src/Fantasma/EntityFramework/IFantasmaDatabase.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace Fantasma.EntityFramework;
+
+[PublicAPI]
+public interface IFantasmaDatabase : IDisposable
+{
+    DbSet<FantasmaJob> Jobs { get; set; }
+    DbSet<FantasmaClusterNode> Cluster { get; set; }
```

Comment:
```suggestion
    DbSet<FantasmaJob> Jobs { get; }
    DbSet<FantasmaClusterNode> Cluster { get; }
```

Only get should suffice no?